languages/elixir/exercises/concept/booleans/lib/rules.ex

```diff
@@ -0,0 +1,10 @@
+defmodule Rules do
+  # TODO: implement the eat_ghost?/2 function
```

Comment:
Do we want to provide a named function stub for the students?

Reply:
meant to, seems to have been an oversight